### PR TITLE
Add ONTOWallet adapter

### PR DESCRIPTION
**Pull request by PieroDaVinci**

### CODE REVIEW

**Comment by PieroDaVinci** [comment]
@jordansexton Hi man, We have made modifications according to the requirements, please check and feedback, thanks.

File: packages/wallets/onto/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    "peerDependencies": {
+        "@solana/web3.js": "^1.44.3"
+    },
+    "dependencies": {
+        "@solana/wallet-adapter-base": "^0.9.9"
+    },
+    "devDependencies": {
+        "@solana/web3.js": "^1.44.3"
+    }
```
Comment (jordansexton) [inline packages/wallets/onto/package.json:37]:
```suggestion
    "peerDependencies": {
        "@solana/web3.js": "^1.50.1"
    },
    "dependencies": {
        "@solana/wallet-adapter-base": "^0.9.12"
    },
    "devDependencies": {
        "@solana/web3.js": "^1.50.1"
    }
```
Just updating these for consistency with the other packages.

File: packages/wallets/onto/src/adapter.ts [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+declare const window: OntoWindow;
+
+export interface OntoWalletAdapterConfig {}
+
+export const OntoWalletName = 'ONTO' as WalletName<'ONTO'>;
+
+export class OntoWalletAdapter extends BaseMessageSignerWalletAdapter {
+    name = OntoWalletName;
+    url = 'https://onto.app/';
```
Comment (jordansexton) [inline packages/wallets/onto/src/adapter.ts:42]:
```suggestion
    url = 'https://onto.app';
```

File: packages/wallets/wallets/src/index.ts [hunk 3]
```diff
@@ -35,4 +35,4 @@ export * from '@solana/wallet-adapter-tokenary';
 export * from '@solana/wallet-adapter-tokenpocket';
 export * from '@solana/wallet-adapter-torus';
 export * from '@solana/wallet-adapter-trust';
-export * from '@solana/wallet-adapter-keystone';
+export * from '@solana/wallet-adapter-keystone';
```
Comment (jordansexton) [inline packages/wallets/wallets/src/index.ts:38]:
```suggestion
export * from '@solana/wallet-adapter-keystone';
export * from '@solana/wallet-adapter-onto';
```

**Review by jordansexton — CHANGES_REQUESTED** [review]
Looks good! Please accept the few nit changes here, and delete the `yarn-error.log` file.

File: packages/wallets/wallets/src/index.ts (hunk 3 above)
Comment (jordansexton) [reply packages/wallets/wallets/src/index.ts:38]: Oh, also add the new onto adapter to the wallets package please!

**Comment by PieroDaVinci** [comment]
@jordansexton Hi,man. I have solved the problem, please check again.

**Comment by PieroDaVinci** [comment]
@jordansexton Hi,man. I have solved the problem, please check again.

File: README.md [hunk 4]
```diff
@@ -189,6 +189,7 @@ You can use the [wallets](https://github.com/solana-labs/wallet-adapter/tree/mas
 | [torus](https://github.com/solana-labs/wallet-adapter/tree/master/packages/wallets/torus)             | Adapter for [Torus](https://tor.us)                   | [`@solana/wallet-adapter-torus`](https://npmjs.com/package/@solana/wallet-adapter-torus)             |
 | [trust](https://github.com/solana-labs/wallet-adapter/tree/master/packages/wallets/trust)             | Adapter for [Trust Wallet](https://trustwallet.com)   | [`@solana/wallet-adapter-trust`](https://npmjs.com/package/@solana/wallet-adapter-trust)             |
 
+
```
[the remark this answers is not in the document]
Comment (jordansexton) [reply README.md:199]: This was marked as resolved, but hasn't been changed.

**Comment by jordansexton** [comment]
Published:
- @solana/wallet-adapter-onto@0.1.0
- @solana/wallet-adapter-wallets@0.18.9

**Pull request merged**